### PR TITLE
chore: standard semantic release rules

### PR DESCRIPTION
**Pull request by FoSix**
## Description

Remove the custom release version calculation rules: feat -> patch, braking -> minor, feat(MAJOR) -> major.

## Motivation and Context

There were no braking changes recently, we can assume the code is major enough not to produce tons of major versions. 
We also need to align the rules for calculating releases with [`pan-os-ansible`](https://github.com/PaloAltoNetworks/pan-os-ansible) project.

## Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes if appropriate.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by michalbil — APPROVED** [review]

**Review by alperenkose — APPROVED** [review]

**Comment by alperenkose** [comment]
Shall we wait for the #128 merge which would be a breaking change before merging this?

**Pull request merged**
